modules/ROOT/pages/update-streams.adoc

```diff
@@ -54,9 +54,11 @@ Switching between streams may introduce regressions or bugs due to skipping upda
 sudo systemctl stop zincati.service
 
 # Perform the rebase to a different stream
+# Supported architectures: aarch64, x86_64
 # Available streams: "stable", "testing", and "next"
+ARCH="$(arch)"
```

Comment:
```suggestion
ARCH="x86_64"
```

I feel like `$(arch)` is more confusing. We should either switch it to a real value or switch `STREAM="testing"` below to `STREAM=$(stream)`.

Reply:
`$(arch)` is the output of the `arch` command...`$(stream) is going to result in an error

Reply:
oh wow. TIL about `/usr/bin/arch`. Sorry for the noise.